services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/AbstractJdbcRegistryTest.java

```diff
@@ -103,7 +103,8 @@ void startDevices(final Vertx vertx) throws IOException, SQLException {
                 properties
         );
         this.registrationAdapter = new RegistrationServiceImpl(
-                DeviceStores.adapterStoreFactory().createTable(vertx, TRACER, jdbc, Optional.empty(), Optional.empty(), Optional.empty())
+                DeviceStores.adapterStoreFactory().createTable(vertx, TRACER, jdbc, Optional.empty(), Optional.empty(), Optional.empty()),
+                new NoOpSchemaCreator()
```

Comment:
why don't we use the SchemaCreator here as well?

Reply:
> why don't we use the SchemaCreator here as well?

I tried this. But that resulted in sporadic, hard-to-analyze test failures, and I'd rather not invest any more time in it at this point.

Reply:
So now we end up with two different approaches being used to create the tables in the DB? I do not think I understand why you introduced `SchemaCreator` in the first place, if we want (need) to also keep the existing mechanism for creating the schemas ...

Reply:
No, you're right. We shouldn't spend any more time on it for now.
@ctron any further comments?